arches/app/templates/views/components/datatypes/resource-instance.htm

```diff
@@ -35,13 +35,19 @@
                             <i class="fa fa-trash"></i>
                         </button>
                     </div>
-                    <div class="rr-table-column" tabindex="0" role="button" style="flex-grow: 1; padding-left: 10px;"
-                        data-bind="
+                    <div class='rr-table-column icon-column'>
+                        <button 
+                            data-bind="
                             click: function() { if($parent.isEditable !== false){self.toggleSelectedResource($data)}; },
                             clickBubble: false,
                             onEnterkeyClick, onSpaceClick," 
-                        style="cursor: pointer;">
+                            style="cursor: pointer;">
+                            <i class="fa fa-gear"></i>
```

Comment:
This has meaning, so let's add an `aria-label`. Maybe there is a suitable string already in javascript.htm like `edit`?

Reply:
requested changes in latest